src/orange/OrangeParams.cc

```diff
@@ -17,27 +17,22 @@
 #include "corecel/cont/Array.hh"
 #include "corecel/cont/Range.hh"
 #include "corecel/data/Collection.hh"
-#include "corecel/data/Ref.hh"
+#include "corecel/data/CollectionBuilder.hh"
 #include "corecel/io/Logger.hh"
 #include "corecel/io/ScopedTimeLog.hh"
 #include "corecel/io/StringUtils.hh"
 
 #include "Data.hh"
 #include "Types.hh"
-#include "construct/SurfaceInput.hh"
-#include "construct/SurfaceInserter.hh"
-#include "construct/VolumeInput.hh"
-#include "construct/VolumeInserter.hh"
+#include "construct/OrangeInput.hh"
+#include "detail/SurfaceInserter.hh"
+#include "detail/VolumeInserter.hh"
 #include "univ/detail/LogicStack.hh"
 
 #if CELERITAS_USE_JSON
 #    include <nlohmann/json.hpp>
 
-#    include "corecel/cont/Array.json.hh"
-#    include "corecel/cont/Label.json.hh"
-
-#    include "construct/SurfaceInputIO.json.hh"
-#    include "construct/VolumeInputIO.json.hh"
+#    include "construct/OrangeInputIO.json.hh"
```

Comment:
I feel like I'm being obtuse here, but I don't see where the `from_json()` functions are being called in this translation unit.